app/serializers/move_serializer.rb

```diff
@@ -25,15 +25,16 @@ class MoveSerializer < ActiveModel::Serializer
   has_many :court_hearings, serializer: CourtHearingSerializer
   belongs_to :allocation, serializer: AllocationSerializer
 
-  INCLUDED_ATTRIBUTES = {
-    person: %i[ethnicity gender],
-    from_location: [],
-    to_location: [],
-    documents: [],
-    prison_transfer_reason: [],
-    court_hearings: [],
-    allocation: [],
-  }.freeze
+  INCLUDED_ATTRIBUTES = %w[
+    person.ethnicity
+    person.gender
+    from_location
+    to_location
+    documents
+    prison_transfer_reason
+    court_hearings
```

Comment:
Seems like we don't need `court_hearings`, so could remove here.

Reply:
we are showing `court_hearings` in the single view endpoint though not the list one

Reply:
Ah, bollocks. You're right - it's referenced in two places inside `moves_controller.rb`. Harrumph.